announcements/2020-Week-37/apps.md

```diff
@@ -0,0 +1,17 @@
+# :zap: This Week in Provider-Independent Releases (September 10, 2020):zap:
+
+**Giant Swarm Catalog (Managed Apps)**
+
+1. [Kong app v0.9.0](https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#090---2020-08-25) upgrades default Kong version to 2.1.3, updates other dependencies, syncs with upstream chart 1.8.0.
+2. [Kong app v0.9.1](https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#091---2020-09-07) Split image repository value to allow switching registry i.e. instead of default Quay use Alibaba by default in China.
```

Comment:
```suggestion
2. [Kong app v0.9.1](https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#091---2020-09-07) splits image repository value to allow switching registry i.e. instead of default Quay use Alibaba by default in China.
```